tests/test-util/src/math.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { degreesToRadians, radiansToDegrees } from "@hpcc-js/util";
+import { expect } from "chai";
+
+describe("math", function () {
+    it("degreesToRadians", function () {
+        expect(degreesToRadians(0)).to.equal(0);
+    });
+    it("radiansToDegrees", function () {
+        expect(radiansToDegrees(0)).to.equal(0);
+    });
```

Comment:
Add more tests (all functions should be well covered) - try and focus on the boundary conditions (like 0, 2PI etc).